.travis.yml

```diff
@@ -14,19 +14,20 @@ cache:
 branches:
   only:
     - master
-    - /v\d+\.\d+\.\d+$/   # Milestone branch pattern for vM.m.p
+    - /v\d+\.\d+\.\d+$/ # Milestone branch pattern for vM.m.p
     - /^feature\/(.+)$/
 
 jobs:
   include:
     - stage: Unit Tests
       if: type = pull_request
-      script: yarn verify
+      script: yarn verify && cd packages/docs && yarn test:build && cd ../..
```

Comment:
Why not make the `verify` script in the docs package be that `test:build` above?